src/axom/sidre/core/View.cpp

```diff
@@ -46,12 +42,14 @@ std::string View::getPath() const
  */
 std::string View::getPathName() const
 {
-  if (getPath().length() < 1)
+  const auto p = getPath();
```

Comment:
``p`` seems short for a variable name.

Reply:
True, but it is a three line function, and it seemed like ``path`` wouldn't necessarily add any additional information.

Reply:
You could do it the old-style Fortran way and eliminate the vowels; i.e., "pth"... :-)

Reply:
When you only have 8 characters.  The current name size limit is much larger but old habits die hard.

Reply:
Should I change it? Or are people ok with ``p`` in this case?

I'm ok with either option.

Reply:
I'm fine with it in this case.  I commented because of the style guide:

    2.2 Tersely abbreviated or cryptic names should be avoided. However, common acronyms   and jargon that are well understood by team members and users may be used.

Reply:
I updated the variable name to ``path`` in both files.